mozdownload/scraper.py

```diff
@@ -816,6 +816,28 @@ def detect_platform(self):
 
         return platform
 
+    def is_build_dir(self, dir):
+        """Return whether or not the given dir contains a build."""
+
+        url = urljoin(self.base_url, self.build_list_regex, dir)
+
+        if self.application in MULTI_LOCALE_APPLICATIONS \
+                and self.locale != 'multi':
+            url = urljoin(url, self.locale)
```

Comment:
So specific locales here only en-US is available in a sub folder. The binary including all other locales is in the same folder. That's why the locale part has to be added again. Keep in mind it works only for en-US at the moment. :)